libminifi/src/controllers/SSLContextService.cpp

```diff
@@ -112,6 +112,22 @@ const core::Property SSLContextService::ClientCertKeyUsage(
         ->build());
 #endif  // WIN32
 
+namespace {
+bool is_valid_and_readable_path(const std::filesystem::path& path_to_be_tested) {
+  std::ifstream file_to_be_tested(path_to_be_tested);
+  return file_to_be_tested.good();
+}
+
+#ifdef WIN32
+std::string getCertName(const utils::tls::X509_unique_ptr& cert) {
+  const size_t BUFFER_SIZE = 256;
+  char name_buffer[BUFFER_SIZE];
+  X509_NAME_oneline(X509_get_subject_name(cert.get()), name_buffer, BUFFER_SIZE);
```

Comment:
according to [this](https://www.openssl.org/docs/man3.1/man3/X509_NAME_oneline.html) the usage of `X509_NAME_oneline` is "strongly discouraged in new applications" although not yet deprecated

Reply:
Updated in 67eb58112fe948432498c39cae18aec78251fd22